base/linalg/generic.jl

```diff
@@ -174,6 +174,12 @@ end
 cond(x::Number) = x == 0 ? Inf : 1.0
 cond(x::Number, p) = cond(x)
 
+#Skeel condition numbers
+condskeel(A::AbstractMatrix, p::Real=Inf) = norm(abs(inv(A))*abs(A), p)
+condskeel{T<:Integer}(A::AbstractMatrix{T}, p::Real=Inf) = norm(abs(inv(float(A)))*abs(A), p)
+condskeel(A::AbstractMatrix, x::AbstractVector, p::Real=Inf) = norm(abs(inv(A))*abs(A)*abs(x), p)
+condskeel{T<:Integer}(A::AbstractMatrix{T}, x::AbstractVector, p::Real=Inf) = norm(abs(inv(float(A)))*abs(A)*abs(x), p)
```

Comment:
@jiahao, why were the `Integer` versions needed, since `inv(A)` already converts to float?

Note also that `abs(inv(A))*abs(A)*abs(x)` should be `abs(inv(A))*(abs(A)*abs(x))` in order to perform matvec and not matmul operations.

See also the discussion in #17302.